distribution/docker/alpine/Dockerfile

```diff
@@ -3,12 +3,12 @@ FROM alpine:3.14 AS builder
 WORKDIR /vector
 
 COPY vector-*-unknown-linux-musl*.tar.gz ./
-RUN tar -xvf vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz --strip-components=2
+RUN tar -xvf "vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz --strip-components=2"
```

Comment:
I don't think the quotes are correct here? Shouldn't it be:
```suggestion
RUN tar -xvf "vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz" --strip-components=2
```

Reply:
Fixed f1c132ef8

Reply:
See my last comment, these quotes are actually undesirable 🙈

Reply:
_Feel like you're just trolling me now_ :P

Reply:
My brain is trolling myself first and foremost!
Those quotes did smell weird at first sight, but then got distracted by their positioning! >_<

Reply:
Yeah - they're _extra_ wrong regardless. I'll confirm your expectations and see that the quoting breaks things here.

Reply:
I confirmed ìt (and remembered from my bash-fu) before commenting haha:
```
$ ls -d "*"
ls: cannot access '*': No such file or directory
```

Reply:
Ah yes, that's an idiomatic fix indeed!
It protects the _variable expansion_ but leaves the _globbing_ intact. 💯

Reply:
_Just need to keep making the quoted section smaller_ 🤌

Reply:
Fixed for real ™️ e2c3e7e57